src/screens/TopicPageScreen/MemoizedListComponent.js

```diff
@@ -66,6 +68,8 @@ const RenderListFeed = (props) => {
     showScoreButton
   } = useFeed();
   const {handleUserName} = useWriteComment();
+  const {followUnfollowTopic} = usePostHook();
+  const route = useRoute();
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The `followUnfollowTopic` function is being imported from `usePostHook`. Make sure that this function is properly tested and handles all edge cases, as it seems to be related to important functionality (following/unfollowing a topic).



<!-- This is an auto-generated comment by OSS CodeRabbit -->